dhctl/pkg/log/logger.go

```diff
@@ -40,13 +40,21 @@ func init() {
 }
 
 func InitLogger(loggerType string) {
```

Comment:
This is function for initialize logger, but i don't see  outstream param

Reply:
https://github.com/deckhouse/deckhouse/pull/5777/commits/85548169f82ab07005c00e5012bc4c5676f05d64#diff-b3ce967f2ebde578ed3a1a2a03e5e36e200aeb2f3cc1bb81efd46db1baa25ceaR50

I created InitLoggerWithOptions alternative init function and InitLogger which uses InitLoggerWithOptions internally.

But if you say it is ok to add options right into InitLogger, then I will do it.